src/Identity/Controllers/AccountsController.cs

```diff
@@ -69,6 +107,25 @@ IDataProtectorTokenFactory<RegistrationEmailVerificationTokenable> registrationE
         _referenceEventService = referenceEventService;
         _featureService = featureService;
         _registrationEmailVerificationTokenDataFactory = registrationEmailVerificationTokenDataFactory;
+
+        if (CoreHelpers.SettingHasValue(globalSettings.KdfDefaultHashKey))
+        {
+            _defaultKdfHmacKey = Encoding.UTF8.GetBytes(globalSettings.KdfDefaultHashKey);
+        }
+    }
+
+    static AccountsController()
+    {
+        // Add some random defaults as well
+        var random = new Random();
+        for (var i = 0; i < 2; i++)
+        {
+            _defaultKdfResults.Add(new()
+            {
+                Kdf = KdfType.PBKDF2_SHA256,
+                KdfIterations = random.Next(100_000, 1_000_000),
+            });
```

Comment:
Does the randomness provide any additional protection? Doesn't it provide information if a given email corresponds to the same index position (of one of the random KDF iterations) across n requests. The caller would see the information change across n requests, right? If so, then they could extrapolate that the user doesn't exist. 

Per a discussion with @justindbaur , the random values would be different across each identity node as well which I believe would reduce the effectiveness of including random data. 

Further, if we are going to include random data, we probably should normalize it to 100k increments (as recommended in our [docs](https://bitwarden.com/help/kdf-algorithms/#pbkdf2)) so that the KDF iterations aren't something like "159,549" which it is extremely unlikely a normal person would pick.

Reply:
Yea, I didn't think about the various nodes in a load balanced setup. I think we should probably just revert the random values then.

Reply:
Done